.github/workflows/reusable-release-policy-rego.yml

```diff
@@ -0,0 +1,49 @@
+name: Build and release a Kubewarden policy written in Rego
+
+on:
+  workflow_call:
+    inputs:
+      oci-target:
+        type: string
+        required: true
+      policy-type:
+        description: "Type of Rego policy; either 'gatekeeper' or 'opa'"
+        type: string
+        required: true
```

Comment:
This is not used inside of the action :thinking:

Reply:
Woops, indeed. Removed in https://github.com/kubewarden/github-actions/pull/15/commits/0187fdb3b5ff7393a04a2a092e817442b8a857d5.